src/patternfly/components/Content/content.scss

```diff
@@ -327,7 +327,7 @@
     grid-template-columns: 1fr;
 
     @media screen and (min-width: $pf-v6-global--breakpoint--sm) {
-      grid-template: auto / auto 1fr;
+      grid-template: auto / 12ch 1fr;
       grid-row-gap: var(--#{$content}--dl--RowGap);
       grid-column-gap: var(--#{$content}--dl--ColumnGap);
```

Comment:
The column gap for `<dl>`'s is 2xl and the row gap is md. Do we want to update that? Some thoughts:
* In figma, the description list default column gap is sm, and row gap is lg. The compact description list column gap is sm, and row gap is md
* Currently our description list component's column and row gaps match what's in figma
* We have gap spacer tokens here - https://github.com/patternfly/patternfly/blob/bdd2cc6ea5d1d66773e74935441880783c121df1/src/patternfly/base/tokens/tokens-default.scss#L467-L474
  * For the default `<dl>` we could use `gap--text-to-element` and `gap--group-to-group--vertical`? I'm not 100% those are the right tokens though.
  * There are no compact variants of gap tokens.
  * If we update to gap spacer tokens, should we also update the description list component?